src/store/store.js

```diff
@@ -45,5 +45,6 @@ const store = createStore(
 
 
 store.dispatch(runTheApp())
+store.dispatch(fetchPosts())
```

Comment:
as you remember, here we add all listeners - fetch posts name is not relevant here
in future we will have one action with all listeners!

Reply:
Ok, I will find a better place for this event listener